nodes/ui_colour_picker.html

```diff
@@ -40,9 +41,11 @@
             $("#node-input-format").change(function() {
                 if ($(this).val() === "hex") {
                     $("#node-alpha-control").hide();
+                    $("#node-lightness-control").hide();
```

Comment:
Not sure we need to hide this one - When in hex mode I think we still want to be able to control brightness